com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfSensors.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace Unity.Robotics.UrdfImporter
+{
+    public class UrdfSensors:MonoBehaviour
```

Comment:
coding style: space after colon :
Also, Is this class used anywhere? what is the purpose of it?

Reply:
https://github.com/Unity-Technologies/URDF-Importer/blob/vidur/tf/com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfSensorsExtensions.cs